sklearn/linear_model/_logistic.py

```diff
@@ -1090,12 +1094,12 @@ class LogisticRegression(LinearClassifierMixin,
         .. versionadded:: 0.17
            *class_weight='balanced'*
 
-    random_state : int, RandomState instance, default=None
+    random_state : int, RandomState instance or None, default=None
```

Comment:
We haven't decided what to do here:

XREF: https://github.com/scikit-learn/scikit-learn/issues/17748